config.json

```diff
@@ -638,6 +638,20 @@
       "unlocked_by": "rotational-cipher",
       "uuid": "d36ce010-210f-4e9a-9d6c-cb933e0a59af"
     },
+    {
+      "core": false,
+      "difficulty": 5,
+      "slug": "grep",
+      "topics": [
+        "pattern_matching",
+        "strings",
+        "files",
+        "filtering",
+        "searching"
+      ],
+      "unlocked_by": "robot-name",
```

Comment:
Feel free to suggest a different difficulty or a different exercise for it to be unlocked by! 🙂 

I thought 5 might be a good difficulty since it has some complexity but not too much. And `robot-name` is also about strings and pattern matching (it has difficulty 5 as well) so I thought it might be a good fit for unlocked by 🙂

Reply:
I agree - this seems like a good fit!